modules/utxo-lib/test/bitgo/legacysafe/index.ts

```diff
@@ -34,9 +34,9 @@ describe('legacy safe scripts creation', function () {
   const compressedPubKeys = compressedKeyPairs.map((keyPair) => keyPair.publicKey);
 
   it('throws error for compressed public keys', function () {
-    assert.throws(
+    assert.doesNotThrow(
```

Comment:
the second arg here is not required anymore

you could add a test with a empty buffer or something